common/utils/src/main/resources/error/error-classes.json

```
@@ -3177,6 +3177,11 @@
           "<variableName> is a VARIABLE and cannot be updated using the SET statement. Use SET VARIABLE <variableName> = ... instead."
         ]
       },
+      "TABLE_NOT_SUPPORTED_STATISTIC" : {
+        "message" : [
+          "Cannot gather statistics for the table <tableName> because it does not the feature."
```

The error class already has the words about unsupported feature, see:
```json
  "UNSUPPORTED_FEATURE" : {
    "message" : [
      "The feature is not supported:"
    ],
```
Please, remove the tail:
```suggestion
          "Gather statistics for the table <tableName>."
```

---

Done.